### ISSUE TITLE
Need to add another caveat

### ISSUE THREAD

**Issue by vasishth** [vasishth/SchoknechtVasishthInterference#37]
At no point did I see a caveat that syntactic interference might still be in play. I have to add such a caveat somewhere.

**Comment by PSchoknecht** [comment]
Isn't the caveat in the general discussion enough?

\Copy{gendisccaveat}{Thus, it seems that, cross-linguistically, syntax does generally play a central role in building incremental structure and in completing dependencies.}\label{gendisccaveat} \Copy{gendisccaveat2}{\textcolor{blue}{Moreover, it is entirely possible that a stronger syntactic manipulation than the one we used ends up showing a syntactic interference effect of the type that \citet{vandyke07} originally reported. Indeed, our own findings would need to be replicated, ideally by an independent research group, if we want to be sure that in the present design there is no evidence for syntactic interference. If such a replication attempt is carried out, it would also be useful to conduct an lab-based self-paced reading study to investigate our speculation earlier that the absence of syntactic interference in our online SPR study may have been due to participants adopting a good-enough processing strategy, leading to only semantic interference being detectable.}}\label{gendisccaveat2}
